Codeinwp/wp-maintenance-mode
# Add Optimole checkbox in wizard

Closes https://github.com/Codeinwp/optimole-service/issues/662.

- Add a checkbox in the wizard that if checked, going to the next step will install and activate Optimole.
- Replace text
- Replace the right top corner icon with an `X` one
- Remove the Otter banner from the wizard
- Change the style of the selected template

<img width="1000" alt="Screenshot 2023-02-15 at 21 55 10" src="https://user-images.githubusercontent.com/39873395/219138505-de17b6b3-7387-4a83-bd0d-3679f5b9e83a.png">


### Test instructions
- The test instance must not have installed the plugin before for the wizard to show, or it can be simulated with `wp option update wpmm_fresh_install 1`
- If the Optimole checkbox is checked, and the `Continue` button is clicked, it should install and activate both Otter and Optimole
- If the Optimole checkbox is checked, and the `I don't want a template` button is clicked, it should install and activate just Optimole
- If the checkbox is unchecked it should behave like before.
